packages/atst/package.json

```diff
@@ -6,6 +6,23 @@
   "types": "src/index.ts",
   "module": "dist/index.cjs",
   "license": "MIT",
+  "exports": {
+    ".": {
+      "types": "./src/index.ts",
+      "main": "./dist/index.js",
+      "module": "./dist/index.cjs"
+    },
+    "./cli": {
+      "types": "./src/cli.ts",
+      "main": "./dist/cli.js",
+      "module": "./dist/cli.cjs"
+    },
+    "./react": {
+      "types": "./src/react.ts",
+      "main": "./dist/react.js",
+      "module": "./dist/react.cjs"
+    }
+  },
```

Comment:
this makes it so the imports work like this

import { readAttestation } from '@eth-optimism/atst'
import { readAttestation } from '@eth-optimism/react'